source/driver_run.cpp

```diff
@@ -2,86 +2,134 @@
 #include "module_cell/module_neighbor/sltk_atom_arrange.h"
 #include "module_hamilt_pw/hamilt_pwdft/global.h"
 #include "module_io/input.h"
+#include "module_io/para_json.h"
 #include "module_io/print_info.h"
 #include "module_io/winput.h"
 #include "module_md/run_md.h"
-#include "module_io/para_json.h"
 
+#ifdef __LCAO
+#include "module_beyonddft/esolver_lrtd_lcao.h"
```

Comment:
this line should not appear in driver_run.cpp, everything should be contained in esolver